rust/scx_loader/src/main.rs

```diff
@@ -467,10 +467,11 @@ async fn start_scheduler(
                         log::debug!("Received cancellation signal");
                         // Send SIGINT
                         if let Some(child_id) = child.id() {
-                            nix::sys::signal::kill(
-                                nix::unistd::Pid::from_raw(child_id as i32),
-                                nix::sys::signal::SIGINT,
-                            ).context("Failed to send termination signal to the child")?;
+                            let ret = unsafe { libc::kill(child_id as i32, libc::SIGINT) };
```

Comment:
Hmm looks less convenient, as `nix` provides good "safe" wrapper around `libc`.

IMHO I'd want more usage of "safe" code

Reply:
hmmm yes and no, ultimately nix calls libc::kill under the hood, there is a tiny safety net for the arguments types. nix strength relies more on interface unification across platforms IMHO but if scx folks wants me to close it I won t argue :)

Reply:
Not huge on safe/unsafe until it becomes problematic, but wrt/ maintainability I'm all for using crates to make things being done simpler.

Consistency (doing the same thing in more places) is good if not great, but I think the overall direction should be "use more crates and make things simpler" not "do things in harder manners to increase consistency".

Reply:
Fair point, note I m not against nix personally, I contribute to it. Anyhow, let s close it then :)